File: mobile/SisCtrlDiabetes/src/com/example/sisctrldiabetes/classes/Food.java
package com.example.sisctrldiabetes.classes;

import com.google.gson.annotations.SerializedName;

public class Food {
	
	@SerializedName(value = "Id")
	int _id;
	
	@SerializedName(value = "Name")
	String _name;
	
	@SerializedName(value = "Weight")
    Double _weight;
	
	@SerializedName(value = "Carbohydrate")
    Double _carb;
	
	@SerializedName(value = "Fiber")
    Double _fiber;
    
    public Food(){

    }
    // constructor
    public Food(int id, String name, Double weight, Double carb, Double fiber){
        this._id = id;
        this._name = name;
        this._weight = weight;
        this._carb = carb;
        this._fiber = fiber;
    }
    
    public int getId() {
		return _id;
	}
	public void setId(int _id) {
		this._id = _id;
	}
	public String getName() {
		return _name;
	}
	public void setName(String _name) {
		this._name = _name;
	}
	public Double getWeight() {
		return _weight;
	}
	public void setWeight(Double _weight) {
		this._weight = _weight;
	}
	public Double getCarb() {
		return _carb;
	}
	public void setCarb(Double _carb) {
		this._carb = _carb;
	}
	public Double getFiber() {
		return _fiber;
	}
	public void setFiber(Double _fiber) {
		this._fiber = _fiber;
	}

    

}

File: webservice/ws/BusinessLayer/DTO/Food.vb
Public Class Food

    Private _id As Integer
    Public Property Id() As Integer
        Get
            Return _id
        End Get
        Set(ByVal value As Integer)
            _id = value
        End Set
    End Property

    Private _name As String
    Public Property Name() As String
        Get
            Return _name
        End Get
        Set(ByVal value As String)
            _name = value
        End Set
    End Property

    Private _weight As Double
    Public Property Weight() As Double
        Get
            Return _weight
        End Get
        Set(ByVal value As Double)
            _weight = value
        End Set
    End Property

    Private _carb As Double
    Public Property Carbohydrate() As Double
        Get
            Return _carb
        End Get
        Set(ByVal value As Double)
            _carb = value
        End Set
    End Property

    Private _fiber As Double
    Public Property Fiber() As Double
        Get
            Return _fiber
        End Get
        Set(ByVal value As Double)
            _fiber = value
        End Set
    End Property

    'Private _usuCreate As String
    'Public Property UsuCreate() As String
    '    Get
    '        Return _usuCreate
    '    End Get
    '    Set(ByVal value As String)
    '        _usuCreate = value
    '    End Set
    'End Property

    'Private _dateCreate As Date
    'Public Property DateCreate() As Date
    '    Get
    '        Return _dateCreate
    '    End Get
    '    Set(ByVal value As Date)
    '        _dateCreate = value
    '    End Set
    'End Property

    'Private _usuUpdate As String
    'Public Property UsuUpdate() As String
    '    Get
    '        Return _usuUpdate
    '    End Get
    '    Set(ByVal value As String)
    '        _usuUpdate = value
    '    End Set
    'End Property

    'Private _dateUpdate As Date
    'Public Property DateUpdate() As Date
    '    Get
    '        Return _dateUpdate
    '    End Get
    '    Set(ByVal value As Date)
    '        _dateUpdate = value
    '    End Set
    'End Property
End Class

